posts/deploying-an-openshift-application-in-aws-govcloud.md

```diff
@@ -0,0 +1,120 @@
+
+#"Offline" Root Certificate Authority
+We had an additional security requirement to house the root certificate authority in a completely isolated and “offline” environment. This needed to be a separate account from the account running the application. More user management was required because RBAC policies needed to be applied to both accounts.
+
+Due to stricter access requirements, engineers were not given access to this new account. That means training was required to launch an EC2 instance, connect via the AWS Console, create a Root Certificate Authority (CA), sign certificates from the application account correctly, change the CA certificate path length variable, and then package the Root Certificate in an encrypted WORM S3 bucket.
+
+Management of the Root CA must be done manually to update the CRL or revoke certificates. Any changes to this component have impacts downstream.
+
+_Tip #2: Create your root certificate account early so delays are not caused on the infrastructure setup due to missing certificates._
+
+#Route53 in GovCloud
+Public-hosted zones are not allowed in GovCloud, but were needed to do DNS validation for our ACME provider (i.e. Let’s Encrypt). In this case, we needed yet another commercial AWS account. Similarly to the offline Root CA, we needed more user management to apply the RBAC policies to this account as well.
+
+![Route53 in GovCloud](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2021/12/route53.png)
+
+_Tip #3: Create your public Route53 account early so delays are not caused on the infrastructure setup due to lack of DNS validation._
+
+#OpenShift 4.x CoreOS
+RedHat strongly recommends running CoreOS as the “operating system” for OpenShift nodes. RedHat also does not recommend manually modifying the base operating system and suggests changes should be done through a privileged pod that mounts directly to the host. The preferred approach is to use operators.
+
+Operators can be installed to handle security and compliance tasks, log forwarding, and other functions that a Linux agent would normally handle. Custom operators can be built if absolutely necessary. For security purposes, we disabled root access to the nodes as well.
+
+_Tip #4: If you need agents installed on your infrastructure nodes, verify operators exist with the needed functionality instead._
+
+#Custom Images
+Custom images were required since CIS hardened images cost additional money and we were already paying for RedHat licenses. Standard RedHat Enterprise Linux (RHEL) licenses are not hardened. We added the following items to build the custom images:
+- [OpenSCAP](https://www.open-scap.org/) to validate and apply security changes
+- FIPS module
+- Amazon SSM for terminal access
+- SIEM vendor tools such as TrendMicro, Filebeat, Auditbeat, and Nessus
+
+_Tip #5: Use a tool like [Packer](https://www.packer.io/) to build custom images for auditability._
+
+#FedRAMP High
```

Comment:
Moving this section higher throws off your tips, but during the context section and up until here I have no clue what FedRAMP High is other than some set of standards

Reply:
Maybe move this first paragraph up to explain what fedramp is. Then start this section at "Several AWS services"